Orange/data/table.py

```diff
@@ -126,21 +126,23 @@ def set_weights(self, weight):
             but only if those values are all numbers and are not NA/NaN.
         If a sequence of (non-NA/NaN) numbers, set those values as the sequence.
         """
-        # TODO: handle NAs
```

Comment:
I might prefer here a check:
```py
if np.isnan(np.sum(weight)):
    raise ValueError
```
What kind of workflow could result in nan weight values that I'd want filled with median (or 1, depending on the type of object I pass) _silently_?